cmd/internal/pipeline/validate.go

```diff
@@ -0,0 +1,104 @@
+package pipeline
+
+import (
+	"context"
+	"fmt"
+	"github.com/go-git/go-git/v5"
+	"github.com/go-git/go-git/v5/plumbing"
+	"github.com/open-policy-agent/conftest/output"
+	runner2 "github.com/open-policy-agent/conftest/runner"
+	"github.com/spf13/afero"
+	"os"
+)
+
+//ValidatePipeline ensures that the pipeline file meets the policy requirements provided
+func ValidatePipeline(fpath string, policyDir string, policyRepo string, ref string, namespace string, outputFormat string) {
+
+	// Verify that the pipeline file exists.
+	exists, err := afero.Exists(AFS, fpath)
+	if err != nil {
+		os.Exit(1)
+	}
+	if !exists {
+		fmt.Printf("Specified pipeline file: %s does not exist.\n", fpath)
+		os.Exit(1)
+	}
+
+	// Get our pipeline name.
+	pipelineName, err := GetPipelineName(fpath)
+	if err != nil {
+		fmt.Printf("Erorr getting pipeline name: %s\n", err)
+		os.Exit(1)
+	}
+
+	// Create temporary work directory.
+	workDir, err := afero.TempDir(AFS, afero.GetTempDir(AFS, ""), "ec-work-")
+	if err != nil {
+		fmt.Println("Unable to create temporary directory for policy checkout.")
+		os.Exit(1)
+	}
+
+	// Checkout policy repo into work directory.
+	_, err = git.PlainClone(workDir, false, &git.CloneOptions{
+		URL:           policyRepo,
+		Progress:      nil,
+		ReferenceName: plumbing.NewBranchReferenceName(ref),
+		SingleBranch:  true,
+	})
+	if err != nil {
+		fmt.Printf("Error checking out polices: %s\n", err)
+		os.Exit(1)
+	}
+
+	policyPath := fmt.Sprintf("%s/%s", workDir, policyDir)
+
+	// Ensure that the specified policy path exists in our cloned repo
+	exists, err = afero.Exists(AFS, policyPath)
+	if !exists {
+		fmt.Println("Specified policy directory does not exist in policy repo:", policyDir)
+		os.Exit(1)
+	}
+	if err != nil {
+		fmt.Println("Error determining if specified policy directory exists.")
+		os.Exit(1)
+	}
+
+	// Create our policy path arrays
+	var policyPaths = []string{policyPath}
+
+	// Create data dir
+	dataDir := fmt.Sprintf("%s/data/", workDir)
+	_ = AFS.MkdirAll(dataDir, 0755)
+	err = afero.WriteFile(AFS, fmt.Sprintf("%s/data/data.json", workDir), []byte("{\"config\":{}}\n"), 0777)
+	if err != nil {
+		os.Exit(1)
+	}
+
+	// Create our data path array
+	var dataPaths = []string{dataDir}
+
+	// Create our conftest runner
+	var runner = runner2.TestRunner{
+		Policy:    policyPaths,
+		Data:      dataPaths,
+		Namespace: []string{namespace},
+		NoFail:    true,
+		Output:    outputFormat,
+	}
+	results, err := runner.Run(context.Background(), []string{fpath})
+
+	// Output the conftest runner results
+	outputter := output.Get(runner.Output, output.Options{
+		Tracing:            runner.Trace,
+		NoColor:            false,
+		SuppressExceptions: runner.SuppressExceptions,
+	})
+	fmt.Println("# Input file:", fpath)
+	fmt.Println("# Pipeline name:", pipelineName)
+	fmt.Println("# Policy repo:", policyRepo)
+	fmt.Println("# Git ref:", ref)
+	err = outputter.Output(results)
+	if err != nil {
+		os.Exit(1)
+	}
```

Comment:
We should return the output here and have the `validatePipelineCmd` present it, as we do it in the `eval` command [here](https://github.com/hacbs-contract/ec-cli/blob/6fc1fb44e4ebd468d838471454d1845c9a8066da/cmd/eval.go#L90)